src/main/java/com/thevoxelbox/voxelsniper/config/VoxelSniperConfigLoader.java

```diff
@@ -43,7 +65,73 @@ public void setMessageOnLoginEnabled(boolean enabled) {
     }
 
     /**
-     * Returns maximum size of brushes that LiteSnipers can use.
+     * Return default block material.
+     *
+     * @return default type
```

Comment:
*Return default block material type? I guess

This affects the following docs too.